### PR TITLE
[12.0] add settings export

### PR DESCRIPTION
**Pull request by p-tombez**
Port of https://github.com/akretion/connector-search-engine/pull/37

### CODE REVIEW

File: connector_search_engine/data/ir_cron.xml [hunk 1]
```diff
@@ -27,4 +27,17 @@
...
+    <record forcecreate="True" id="ir_cron_export_settings" model="ir.cron">
+        <field name="name">Search engine: Export settings for all indexes</field>
+        <field eval="False" name="active"/>
+        <field name="user_id" ref="base.user_root"/>
+        <field name="interval_number">1</field>
+        <field name="interval_type">days</field>
+        <field name="numbercall">-1</field>
+        <field eval="False" name="doall"/>
+        <field eval="'se.index'" name="model"/>
+        <field eval="'export_all_settings'" name="function"/>
```
Comment (simahawk) [inline connector_search_engine/data/ir_cron.xml]: this must be migrated to v12

**Review by simahawk — CHANGES_REQUESTED** [review]
tests shall be updated

**Comment by p-tombez** [comment]
Not sure why the pre-commit is failing... I ran `pre-commit run --all` locally and everything is green. Does the checks on Travis are different ?

**Comment by simahawk** [comment]
superseded by #38

**Pull request closed without merging**